src/plugins/intel_gpu/include/intel_gpu/plugin/program_builder.hpp

```diff
@@ -143,6 +149,7 @@ class ProgramBuilder final {
 private:
     static factories_map_t factories_map;
     std::shared_ptr<cldnn::program> m_program;
+    std::shared_ptr<ov::Model> m_model;
```

Comment:
does it  mean that `ov::Model` will live forever after being compiled to `ov::CompiledModel` ?